ansible/install-apps.yml

```
@@ -0,0 +1,34 @@
+---
+
+- name: Install OS-level apps
+  hosts: all
+  become: yes
+  vars:
+  - apt_file: /etc/apt/sources.list.d/google-chrome.list
+  - apt_file_Git: 
+  tasks:
+  - name: Does the Google apt file exist?
+    command: test -f {{apt_file}}
+    register: google_apt_exists
+    ignore_errors: yes
+
+  - name: Add Google Chrome key   
+    shell: echo 'deb [arch=amd64] https://dl.google.com/linux/chrome/deb/ stable main'>/etc/apt/sources.list.d/google-chrome.list
+    when: google_apt_exists.rc == 1
+
+  - name: Add Google Chrome repo
+    copy: content="deb http://dl.google.com/linux/chrome/deb/ stable main" dest={{apt_file}} owner=root group=root mode=644
+    when: google_apt_exists.rc == 1
+
+
+  - name: Update apt cache
+    apt: update_cache=yes
+    when: google_apt_exists.rc == 1
+
+  - name: Install apps
+    apt:
+      name: "{{ packages }}"
```

Most of the modules actually accept a list here. So you can do:
```suggestion
      name:
      - google-chrome-stable
      - git
```
But it's probably better to just put vars in the top of the module.